src/dist/replication/test/meta_test/unit_test/config-test.ini

```diff
@@ -111,3 +111,8 @@ args =
 [duplication-group]
 master-cluster = 1
 slave-cluster  = 2
+cluster_without_address = 3
```

Comment:
`cluster_without_address ` is only for test, right? Better to rename to  `cluster_without_address_for_test ` or some other like that to clarify it.